cooperations/ospp-2025/get_repo_tech_label/topic_generate/ablation-study-only-name.py

```diff
@@ -0,0 +1,24 @@
+from openai import OpenAI
+import json
+
+client = OpenAI(
+    base_url='https://xiaoai.plus/v1',
+    # sk-xxx替换为自己的key
+    api_key='sk-iLwJrfFspiWoNHRESsMNsu1YeYBFuGMsREs9GKj2yEZ8mCx1'
```

Comment:
@PureNatural Do not upload your API key in PR.

Reply:
I have deleted the key in the code.